src/components/OrderBookChart.stories.tsx

```diff
@@ -0,0 +1,249 @@
+  const { baseToken, quoteToken } = orderbookProps
+
+  const { asks = [], bids = [] } = sampleData
+  const normalizedRawData: RawApiData = {
+    asks: asks.map(({ amount, price }) => ({
+      // turn small numbers into wei or decimal-aware units
+      // otherwise get filtered as too small by src/components/OrderBookWidget.tsx#L77
+      volume: amount * 10 ** baseToken.decimals,
+      price,
+    })),
+    bids: bids.map(({ amount, price }) => ({
+      volume: amount * 10 ** quoteToken.decimals,
+      price,
+    })),
+  }
+
+  const processedData = processRawApiData({ data: normalizedRawData, baseToken, quoteToken })
+
+  return { ...orderbookProps, data: processedData }
+}
+
+// data samples
+const [
+  liquidMarket,
+  liquidMarketBigSpread,
+  liquidMarketWhenPricesFall20,
+  liquidMarketWhenPricesFall40,
+  liquidMarketWhenPricesFall60,
+  lowVolumeOverlap,
+  highVolumeOverlap,
+  lowVolumeOverlapWithBigPriceDifference,
+  highVolumeOverlapWithBigPriceDifference,
+  spreadIsAlmostNonExistent,
+  bidsLiquidAsksIlliquid,
+  asksLiquidBidsIlliquid,
+  asksLiquidInTheEdgesBidsIlliquid,
+  bidsLiquidInTheEdgesAsksIlliquid,
+  noBids,
+  noAsks,
+  noBidsAndNoAsks,
+] = sampleDataSet
+
+type SampleProps = SampleData & {
+  baseToken: string
+  quoteToken: string
+  networkId: number
+}
+
+// preprocessing template
+// does what OrderBookWidget does for real data
+const SampleTemplate: Story<SampleProps> = ({
+  name,
+  description,
+  asks,
+  bids,
+  networkId,
+  baseToken: baseTokenSymbol,
+  quoteToken: quoteTokenSymbol,
+}) => {
+  const baseToken = useMemo(() => {
+    return (
+      tokenList.find((token) => token.symbol === baseTokenSymbol || token.address === baseTokenSymbol) ||
+      defaultParams.baseToken
+    )
+  }, [baseTokenSymbol])
+
+  const quoteToken = useMemo(
+    () =>
+      tokenList.find((token) => token.symbol === quoteTokenSymbol || token.address === quoteTokenSymbol) ||
+      defaultParams.quoteToken,
+    [quoteTokenSymbol],
+  )
+
+  const props = useMemo(
+    () =>
+      produceOrderBookProps({
+        sampleData: { asks, bids },
+        orderbookProps: {
+          baseToken,
+          quoteToken,
+          networkId: networkId,
+        },
+      }),
+    [networkId, baseToken, quoteToken, asks, bids],
+  )
+
+  return (
+    <div style={{ height: '95%' }}>
+      <details style={{ lineHeight: 1.5 }}>
+        <summary>{name}</summary>
+        {description}
+      </details>
+      <OrderBookChart {...props} />
+    </div>
+  )
+}
+
+const defaultSampleParams = {
+  ...defaultParams,
+  baseToken: 'BASE',
+  quoteToken: 'QUOTE',
+}
+
+// generic builder for Sample Stories
+const buildSampleStory = (data: SampleData): Story<SampleProps> => {
+  const Story = SampleTemplate.bind({})
+  Story.args = {
+    ...defaultSampleParams,
+    ...data,
+  }
+
+  Story.storyName = data.name
+
+  return Story
+}
+
+export const LiquidMarket = buildSampleStory(liquidMarket)
```

Comment:
Okay, I see why you go manual. Probably if you just export a default object, with the Key being the cammelCase of the example, it would work too

Reply:
No how exports work, can't do `export {...AllExports}`, has to be statical named exports
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export